components/nimbus/src/lib.rs

```diff
@@ -2,945 +2,44 @@
-        log::info!("[Nimbus] Installation Date: {}", installation_date);
-        let update_date = self.get_update_date(db, writer)?;
-        log::info!("[Nimbus] Update Date: {}", update_date);
-        let now = Utc::now();
-        let duration_since_install = now - installation_date;
-        log::info!(
-            "[Nimbus] Days since install: {}",
-            duration_since_install.num_days()
-        );
-        let duration_since_update = now - update_date;
-        log::info!(
-            "[Nimbus] Days since update: {}",
-            duration_since_update.num_days()
-        );
-        if state.targeting_attributes.days_since_install.is_none() {
-            state.targeting_attributes.days_since_install =
-                Some(duration_since_install.num_days() as i32);
-        }
-        if state.targeting_attributes.days_since_update.is_none() {
-            state.targeting_attributes.days_since_update =
-                Some(duration_since_update.num_days() as i32);
-        }
-
-        Ok(())
-    }
-
-    /**
-     * Calculates the active_experiments based on current enrollments for the targeting attributes.
-     */
-    fn update_ta_active_experiments(
-        &self,
-        db: &Database,
-        writer: &Writer,
-        state: &mut MutexGuard<InternalMutableState>,
-    ) -> Result<()> {
-        let enrollments_store = db.get_store(StoreId::Enrollments);
-        let prev_enrollments: Vec<enrollment::ExperimentEnrollment> =
-            enrollments_store.collect_all(writer)?;
-
-        let mut set = HashSet::<String>::new();
-        for ee in prev_enrollments {
-            if let EnrollmentStatus::Enrolled { .. } = ee.status {
-                set.insert(ee.slug.clone());
-            }
-        }
-
-        state.targeting_attributes.active_experiments = set;
-
-        Ok(())
-    }
-
-    fn evolve_experiments(
-        &self,
-        db: &Database,
-        writer: &mut Writer,
-        state: &mut InternalMutableState,
-        experiments: &[Experiment],
-    ) -> Result<Vec<EnrollmentChangeEvent>> {
-        let nimbus_id = self.read_or_create_nimbus_id(db, writer)?;
-        let targeting_helper =
-            NimbusTargetingHelper::new(&state.targeting_attributes, self.event_store.clone());
-        let evolver = EnrollmentsEvolver::new(
-            &nimbus_id,
-            &state.available_randomization_units,
-            &targeting_helper,
-        );
-        evolver.evolve_enrollments_in_db(db, writer, experiments)
-    }
+pub use schema::*;
+pub use targeting::NimbusTargetingHelper;
 
-    pub fn apply_pending_experiments(&self) -> Result<Vec<EnrollmentChangeEvent>> {
-        log::info!("updating experiment list");
-        let db = self.db()?;
-        let mut writer = db.write()?;
+cfg_if::cfg_if! {
+    if #[cfg(feature = "stateful")] {
+        mod behavior;
+        mod client;
+        mod config;
+        mod dbcache;
+        mod updating;
```

Comment:
AFAIK it was there previously for `crate::` imports, but I do agree it's not strictly necessary anymore. 

I'm content with removing it if you'd prefer, but I'd prefer to handle it in a different ticket because it'll result in a lot of import changes.